lib/terraforming/cli.rb

```diff
@@ -224,17 +229,20 @@ def tfstate(klass, tfstate_path)
 
     def tfstate_skeleton
       {
-        "version" => 1,
-        "serial" => 0,
-        "modules" => [
-          {
-            "path" => [
-              "root"
-            ],
-            "outputs" => {},
-            "resources" => {},
-          }
-        ]
+          "version" => 3,
+          "terraform_version" => "0.7.3",
+          "serial" => 0,
+          "lineage" =>  SecureRandom.uuid,
```

Comment:
__[rubocop]__
* Operator `=>` should be surrounded by a single space. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)

<!-- Comment by [SideCI](https://sideci.com) -->